Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -0,0 +1,545 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+__RCSID__ = "$Id"
+
+from stat import *
```

Comment:
I don't think `import *` is at anytime a good idea as we don't know what is imported, but anyway `stat.*` is not used anywhere